app/manage/pods/rules-form/conditional-form-directive/template.tmpl

```diff
@@ -47,8 +47,10 @@
         <i class="fa fa-pencil" ng-click='editCondition()'></i>
         <i class="fa fa-times" ng-click='deleteCondition(index)'></i>
       </span>
+      <!--edit mode -->
+      <button class='button secondary' ng-if="!listMode && editting" ng-click='editCondition()'>+ Save </button>
```

Comment:
yes!

Do you think it's weird that it says save and isn't making a server request?  Should it be "update" ?